aim/ext/task_queue/queue.py

```diff
@@ -17,7 +19,9 @@ def __init__(self, name, num_workers=1, max_backlog=0):
         self._threads = []
         self._shutdown = False
         self._stopped = False
-        atexit.register(self.stop_workers)
+
+        if register_at_exit:
```

Comment:
what if we get rid of `atexit` usage and switch to resource cleanup?

Reply:
Done for RPC queue.